recipes/npstructures/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.2.9" %}
+
+package:
+  name: npstructures
+  version: {{ version }}
+
+source:
+  url: https://github.com/bionumpy/npstructures/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b9e34286ac3bd3ac9ce3fda88e6ae768299c82a4ba8d4dd48a0f358f182b3b48
+
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+
+requirements:
+  host:
+    - python >=3
+    - zlib
+    - pip
+  run:
+    - numpy >=1.20,<1.24
```

Comment:
For the sake of consistency can you add `python >=3` to the run section?

Reply:
well spotted